gui/launcher.h

```diff
@@ -73,20 +96,28 @@ class LauncherDialog : public Dialog {
 	StringArray		_domains;
 	BrowserDialog	*_browser;
 	SaveLoadChooser	*_loadDialog;
-
+	String			_title;
 	String _search;
 
+#ifndef DISABLE_LAUNCHERDISPLAY_GRID
+	ButtonWidget		*_listButton;
+	ButtonWidget		*_gridButton;
+
+	void addChooserButtons();
+	ButtonWidget *createSwitchButton(const Common::String &name, const Common::U32String &desc, const Common::U32String &tooltip, const char *image, uint32 cmd = 0);
```

Comment:
Switch-button and Chooser-Buttons, could use a slightly more specific name, or failing that, at least some comments.

Reply:
I have added a comment to better explain these functions in 84ec5b6.